src/lib/components/Calendar.svelte

```diff
@@ -203,6 +219,29 @@
 	</Modal>
 {/if}
 
+{#if $page.state.showingModal === 'addAttendance'}
+	<Modal on:close={() => history.back()}>
+		{@const eventId = $page.state.eventId}
+		{@const event = events.find((e) => e.id === eventId)}
+		{#if event}
+			<h1>Create attendance</h1>
+			<p>This will allow you to take attendance for this event</p>
+			<form
+				action="/org/{$page.params.id}/club/{$page.params.clubId}/attendance?/create"
+				method="post"
+				use:enhance
+			>
+				{selectedDay.toDate().toISOString()}
+				<input name="date" hidden value={selectedDay.toDate().toISOString()} />
```

Comment:
timezone info might be lost here! transform it to UTC first before passing it into the input.

Reply:
`.toISOString()` will auto convert to utc.